### ISSUE TITLE
Support color blind users better by default

### ISSUE THREAD

**Issue by makew0rld** [makew0rld/amfora#189]
In [this](https://www.youtube.com/watch?v=eT7v2RAlDg4&lc=UgzUCllnqGBLzjUETud4AaABAg) YouTube comment someone said they couldn't tell the difference between a foreign link (`#8700d7`) and a gemini link (`#0087ff`). And indeed, it [fails](https://contrast-ratio.com/#%238700d7-on-%230087ff) WCAG 2.0. I'm not sure if that's the best test since it's more about the background and foreground of text, but it means something.

Other important colors should be checked as well, and compared against other colors in use.

Keep in mind I'm sticking to the 256 xterm colors for the default Amfora colors. A list of those colors is here: https://jonasjacek.github.io/colors/

**Comment by makeworld-the-better-one** [comment]
Amfora already has a config option to display the link URL, and that can be done also by using Tab to browse links. I'll think about your suggestion, but for this issue I'm mostly just interested in seeing if there are better color options.

**Comment by jedahan** [comment]
or maybe using dotted underline / solid underline to indicate visited status

**Comment by makeworld-the-better-one** [comment]
@jedahan this isn't about visited status, but about gemini links vs non-gemini links.

I haven't been able to find a good colour option that has enough contrast against both the blue (`#0087ff`) and the black of the default terminal background though, so underlining might be a good option.

**Comment by makeworld-the-better-one** [comment]
Related: 0245267

**Comment by makeworld-the-better-one** [comment]
The default theme colors should be pretty good now, assuming a black terminal background is used. It's not perfect for a white background, but hopefully #181 can help with that.

**Comment by makeworld-the-better-one** [comment]
Was this change necessary, now that #181 is implemented? It would be necessary if the colorblind user had two similar colors in the terminal color scheme. I'm not sure how often that happens, but it feels overreaching for Amfora to underline things by default just because the user's colors might be close together.

Reopening and adding to the v1.10.0 milestone to think about this more.

**Comment by makew0rld** [comment]
> It would be necessary if the colorblind user had two similar colors in the terminal color scheme.

I think it's reasonable for the user to be in charge of this.
